### ISSUE TITLE
No Response in HomeKit 

### ISSUE THREAD

**Issue by DevSpur** [ebaauw/homebridge-rpi#131]
Hi so I've followed the docs on how to set up a device, I'm running raspberry pi os lite (32-bit).
and the latest version of home bridge, in home bridge the accessorize seems to be running fine but as soon as I open the home app all accessories are not responding or have never responded 

any ideas on this

**Comment by ebaauw** [comment]
That’s an issue with the communication between Homebridge and your Apple device, nothing to do with this (or any other) plugin.  Turn WiFi off and back on on your Apple device.  Power cycle your Apple device.  Double-check with the _Discovery_ app that your Apple device receives the Bonjour announcements by Homebridge.  If not, make sure both are on the same subnet, IGMP snooping is disabled on all routers, access points, network switches.

**Comment by oldskole** [comment]
Hi DevSpur, assuming wifi etc is ok... just a similar situation during soak testing I had where things were working then nothing responded on RPi, tried multiple images/SD cards/Rpi hardware, I did updates!!!! then some of the leds on the Rip outputs looked dim but nothing would respond! (I thought  I had wrecked the pi) Also Initially it looked ok on iOS then try to click on stuff however nothing responded.  But what worked for me was to roll back the latest HomeBridge update from v1.5.0 back to v1.4.1 then things started responding again (working) ie driving inputs & outputs... You will find the update or roll back link on the main or status screen (next to QR code). I hope it works for you. the rollback process is quick and can be undone if it don't work. reboot afterwards! Needless to say I was happy to get it working but it cost me 2days! In my case not sure why latest upgrade screwed things up. I will say though I am really impressed with functionality so still soak testing...
